training/nvidia-bootc/Containerfile

```diff
@@ -123,16 +123,38 @@ RUN if [ "${TARGET_ARCH}" == "" ]; then \
     && ln -s /usr/lib/systemd/system/nvidia-toolkit-firstboot.service /usr/lib/systemd/system/basic.target.wants/nvidia-toolkit-firstboot.service \
     && echo "blacklist nouveau" > /etc/modprobe.d/blacklist_nouveau.conf
 
-ARG INSTRUCTLAB_IMAGE="quay.io/ai-lab/instructlab-nvidia:latest"
+
```

Comment:
I think we should separate this out into a separate Containerfile, If we are going to make the images public by default, we don't want to embed the builders ssh keys into the container.  

Perhaps 
```
cat Containerfile.sshpubkey
args from="quay.io/ai-lab/HARDWARE-bootc"
FROM ${from}

ARG SSHPUBKEY

# The --build-arg "SSHPUBKEY=$(cat ~/.ssh/id_rsa.pub)" option inserts your
# public key into the image, allowing root access via ssh.
RUN set -eu; mkdir -p /usr/ssh && \
    echo 'AuthorizedKeysFile /usr/ssh/%u.keys .ssh/authorized_keys .ssh/authorized_keys2' >> /etc/ssh/sshd_config.d/30-auth-system.conf && \
    echo ${SSHPUBKEY} > /usr/ssh/root.keys && chmod 0600 /usr/ssh/root.keys
```

Reply:
Good point will pull this out